CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
-- [NEW] Add `CloudantClient.metaInformation()`.
+- [NEW] Add `CloudantClient.metaInformation()` to get meta information from the welcome page.
+- [NEW] Add methods for interacting with the replicator's `_scheduler` endpoint:
+  - `CloudantClient.metaInformation()`,
+  - `CloudantClient.schedulerJobs()`,
+  - `CloudantClient.schedulerDocs()`,
+  - `CloudantClient.schedulerDocs(docId)`.
```

Comment:
this should probably be singular:
`schedulerDoc(docId)`

or maybe we could just varargs the `schedulerDocs(String... docIds)` to allow 1 or more specific documents to be fetched, with the no-args getting them all, but the annoying thing then is that we'd have to wrap a single document in a `SchedulerDocsResponse`.

Reply:
I changed it to `schedulerDoc(docId)` in 3946f8e as suggested

Reply:
How does 41debcc look for the `return` part of the javadoc? I'm wary of javadoc-ing the `Response` classes but I could copy the descriptions from the HTTP API if you think it's useful.

Reply:
No, what you've done is sufficient, the method signatures are self-explanatory